.github/workflows/tests.yml

```diff
@@ -0,0 +1,66 @@
+# TODO: Line 20, enable python-version: 3.5
+# TODO: Line 69, enable pytest --doctest-modules
+
+name: Tests
```

Comment:
I find the output of Actions less confusing when line 4 (currently Tests) and line 15 (currently test) match.